examples/simulation/Tutorial2/macros/compare_seed_value.C

```diff
@@ -0,0 +1,20 @@
+int compare_seed_value(TString filename, UInt_t initial_seed) {
+
+  // Open parameter file and get the stored random seed from the
+  // FairBaseParSet parameter container
+  // Compare the random seed with the expected one set during the
+  // transport run
+
+  std::unique_ptr<TFile> myFile( TFile::Open(filename) );
```

Comment:
Prevent potential segfault in line 9:
```suggestion
std::unique_ptr<TFile> myFile( TFile::Open(filename) );
if (!myFile) {
  std::cout << "Could not open file: " << filename << std::endl;
  return 1;
}
```

Reply:
Done.